aws/ec2/main.tf

```diff
@@ -0,0 +1,8 @@
+terraform {
+  required_providers {
+    secrethub = {
+      source = "secrethub/secrethub"
+      version = ">= 1.2.3"
```

Comment:
For top-level modules (where you run `terraform apply`) it's generally advised to use `~>` (patches only) instead of `>=`, to avoid unwanted minor version updates. Do we want to use that here as well?